### PR TITLE
Naming all roles to {model}-{action}

### PR DESCRIPTION
**Pull request by Houbsi**
Like we discussed in https://github.com/eramitgupta/laravel-role-permission/issues/5

Here is the first part of the renaming all the roles to the same structure.

### CODE REVIEW

File: src/database/Seeder/RolePermissionSeeder.php.stub [hunk 1]
```diff
@@ -71,14 +71,14 @@ class RolePermissionSeeder extends Seeder
...
-                'permissions' => ['create-post'],
+                'permissions' => ['post-create'],
             ],
             [
                 'name' => 'User',
                 'email' => 'user@gmail.com',
                 'password' => Hash::make('user'),
                 'roles' => ['user'],
-                'permissions' => ['create-user'],
+                'permissions' => ['user-create],
```
Comment (eramitgupta) [inline src/database/Seeder/RolePermissionSeeder.php.stub:81]: The closing single quote missing a

File: README.md [hunk 2]
```diff
@@ -262,14 +262,14 @@ class RolePermissionSeeder extends Seeder
...
-                'permissions' => ['create-post'],
+                'permissions' => ['post-create'],
             ],
             [
                 'name' => 'User',
                 'email' => 'user@gmail.com',
                 'password' => Hash::make('user'),
                 'roles' => ['user'],
-                'permissions' => ['create-user'],
+                'permissions' => ['user-create],
```
Comment (eramitgupta) [inline README.md:272]: The closing single quote is missing a

File: README.md [hunk 3]
```diff
@@ -161,11 +161,11 @@ getRoles();
 To protect routes based on roles and permissions, you can use the provided middleware. For example, to allow only users with the `user` role and `create-user` permission:
 
 ```php
-Route::group(['middleware' => ['role:user,create-user']], function () {
+Route::group(['middleware' => ['role:user,user-create]], function () {
```
Comment (eramitgupta) [inline README.md:164]: The closing single quote is missing a

File: src/database/Seeder/RolePermissionSeeder.php.stub (hunk 1 above)
Comment (Houbsi) [reply src/database/Seeder/RolePermissionSeeder.php.stub]: fixed

File: README.md (hunk 3 above)
Comment (Houbsi) [reply README.md]: fioxed

**Comment by Houbsi** [comment]
While doing the 3 i found some others. where the ' was missing. My bad!

**Pull request merged**